zephyr/Kconfig

```diff
@@ -33,6 +33,16 @@ config MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
 
 endchoice
 
+config MCUBOOT_IMAGE_VERSION
```

Comment:
how does this corporate with what mcuboot/mcumgr can recognize (and logg to the console)?

Reply:
```
struct image_version {
    uint8_t iv_major;
    uint8_t iv_minor;
    uint16_t iv_revision;
    uint32_t iv_build_num;
};
```